ccdproc/image_collection.py

```diff
@@ -216,6 +223,13 @@ def files(self):
         """
         return self._files
 
+    @property
+    def ext(self):
+        """
+        str,The extension for which all 
```

Comment:
Also, this looks like an incomplete sentence -- maybe `for which all` → `from which the header and data will be read in all files.`?